config/configuration.json

```diff
@@ -440,6 +440,15 @@
       "tags":"tags"
     }
   },
+  "BadValueTagCheck": {
+    "challenge": {
```

Comment:
A nice to have would be a config to pass invalid regex pattern similar to InvalidTagsCheck.